generators/entity-client/templates/angular/src/main/webapp/app/entities/list/no-pagination-template.ejs

```diff
@@ -58,7 +60,10 @@
         }
 <%_ } _%>
 
-        this.<%= entityInstance %>Service.query().subscribe({
+        const queryObject = {
```

Comment:
normal?

Reply:
that was some leftover when I tried to implement filtering also without pagination - and then noticed,it requires some basic work to handle routes there first. ;-)